yesod-test/Yesod/Test.hs

```diff
@@ -204,6 +204,9 @@ module Yesod.Test
     , htmlAllContain
     , htmlAnyContain
     , htmlNoneContain
+    , htmlAllContainPreEscaped
+    , htmlAnyContainPreEscaped
+    , htmlNoneContainPreEscaped
```

Comment:
Is `htmlAllContainRaw` etc better than `htmlAllContainPreEscaped`? `PreEscaped` was used since that's what it's called in the [Blaze docs](https://hackage.haskell.org/package/blaze-markup-0.8.2.8/docs/Text-Blaze-Internal.html#v:preEscapedString)